ext/hal/st/stm32cube/stm32f4xx/drivers/include/stm32f4xx_hal_conf.h

```diff
@@ -50,9 +50,15 @@
 
 /* ########################## Module Selection ############################## */
 /**
-  * @brief This is the list of modules to be used in the HAL driver 
+  * @brief This is the list of modules to be used in the HAL driver
+  *
+  *     Note:
+  *     	HAL_I2C_MODULE_ENABLED had to be disabled due to the HAL
+  *     	redefining I2C_SPEED_STANDARD and I2C_SPEED_FAST when the
+  *     	I2C LL driver was enabled.
+  *
   */
-#define HAL_MODULE_ENABLED  
+#define HAL_MODULE_ENABLED

```

Comment:
I'm saying we should avoid any whitespace fixes.  This is so if/when we do a diff against some updated HAL import we can tell what we actually changed vs spurious diffs related to whitespace.